.github/ISSUE_TEMPLATE/bug-issue.md

```diff
@@ -0,0 +1,24 @@
+## What happened?
+_Describe in detail what went wrong; screenshots, videos, or gifs are strongly encouraged_
+
+## Specs:
+- Device: (e.g. iPhone 8, Macbook)
+- Browser: (e.g. Firefox, IE 11)
+- Test User _(if applicable)_:
+
+## Steps to Reproduce
+- URL:
+- Test User _(if applicable)_:
+
+## Desired behavior
+_Describe in detail what was supposed to happen; reference existing Github issue that contains a solution if possible_
+
+## Acceptance Criteria
```

Comment:
what's the difference between "Acceptance Criteria" and "Definition of Done"?

Reply:
the first answer here in this Stack Exchange thread i think does a great job of laying out the differences https://pm.stackexchange.com/questions/18430/what-is-the-difference-between-acceptance-criteria-and-the-definition-of-done

Reply:
based on that description, maybe reword from "What change needs to be made?" to something like "Describe the desired working behavior" ?